packages/jest/src/plugins/plugin.ts

```diff
@@ -211,23 +213,24 @@ async function buildJestTargets(
 
   let metadata: ProjectConfiguration['metadata'];
   if (options?.ciTargetName) {
-    // Resolve the version of `jest-runtime` that `jest` is using.
-    const jestPath = require.resolve('jest');
-    const jest = require(jestPath) as typeof import('jest');
-    // nx-ignore-next-line
-    const { default: Runtime } = require(require.resolve('jest-runtime', {
-      paths: [dirname(jestPath)],
+    const { default: Runtime } =
       // nx-ignore-next-line
-    })) as typeof import('jest-runtime');
+      requireJestUtil<typeof import('jest-runtime')>('jest-runtime');
 
     const jestContext = await Runtime.createContext(config.projectConfig, {
       maxWorkers: 1,
       watchman: false,
     });
 
+    const jest = require('jest') as typeof import('jest');
```

Comment:
In a separate PR, can we change it so that when we resolve `jest`, we resolve it from the config file's directory?